src/core.ts

```diff
@@ -374,16 +373,6 @@ export const TrackingConsentDetails = t.intersection([
     updated: t.boolean,
     /** Whether or not the UI has been shown to the end-user (undefined in older versions of airgap.js) */
     prompted: t.boolean,
-    /** Transparency Consent (TCF) String */
-    tcf: t.string,
-    /** US Privacy (USP) String */
-    usp: t.string,
-    /** Global Privacy Platform (GPP) String */
-    gpp: t.string,
-    /** Consent Manager View State */
-    viewState: valuesOf(ViewState),
-    /** Airgap Version */
-    airgapVersion: t.string,
```

Comment:
:( I'm actually unsure of why this isn't working already. I started to try debugging it but got waylaid. Looking at https://github.com/transcend-io/main/tree/dev/backend-services/consent-backend/src it seems like it _should_ work. I tried posting a usp string and did see it in the database, but for some reason it wasn't getting returned in the response.
If this is a small bug to fix, then I'd rather keep this in, though I don't know your reason for removing it.

Reply:
it shouldn't/wouldn't work. I shared my thoughts in the Slack thread linked to this PR

Reply:
the issue is airgap.js -- this can't be the type for the return function of getConsent()